app/helpers/messages_helper.rb

```diff
@@ -25,17 +27,25 @@ def format_for_speaking(text)
     ::MarkdownRenderer.render_for_speaking(text)
   end
 
-  def format_for_display(text, append_inside_tag: nil)
```

Comment:
a few lines above this is “from_name_for()” and I think we should remove the “when tool” line from that so that the tool calls are just a single line message rather than two lines